wormchain/Makefile

```diff
@@ -67,6 +67,32 @@ ts-sdk: vue
 run: build/wormchaind
 	./$< start --home build  --log_level="debug"
 
+# get npm packages for contracts/tools
+contracts-tools-deps: contracts/tools/package-lock.json
+	@touch -m ./contracts/tools/node_modules
```

Comment:
In the past I've had a problem with this kind of touch command because if the `node_modules` directory doesn't already exist then this will put a file there, which will cause the subsequent `npm ci` command to fail because `node_modules` isn't a directory.